generate_ldscript.py

```diff
@@ -44,6 +44,8 @@ def parse_arguments(argv):
                        help="Emits a linker script with the scratchpad layout")
     group.add_argument("--ramrodata", action="store_true",
                        help="Emits a linker script with the ramrodata layout")
+    group.add_argument("--freertos", action="store_true",
```

Comment:
Is the intention to have ALL freedom-e-sdk FreeRTOS examples using freertos.default.lds, or just the FreeRTOSPMP one? If the former, you will also need to change freedom-e-sdk makefile to reflect this for all freeRTOS example. If the later, rename the tag from "freertos" to "securefreertos", plus you also need to change freedom-e-sdk makefile

Reply:
The script was design for FreeRTOSPMP, but we plan to use it for all freeRTOS, in order to avoid the change in metal.default.lds are required later on (by FreeRTOS standard or refactoring). So I think we are more in the first case, change on freedom-e-sdk makefile are coming.